source/solvers/heat_transfer_assemblers.cc

```diff
@@ -399,9 +402,17 @@ HeatTransferAssemblerRobinBC<dim>::assemble_matrix(
                   for (unsigned int q = 0; q < scratch_data.n_faces_q_points;
                        ++q)
                     {
+                      Point<dim> quadrature_point_on_face;
+                      for (unsigned int d = 0; d < dim; d++)
+                        quadrature_point_on_face[d] =
+                          scratch_data.quadrature_points[q][d];
```

Comment:
Why do you need to make a local copy of the points? why not just juse teh points directly?